src/Service/EventLog/Models/EventResponse.cs

```diff
@@ -4,6 +4,8 @@ namespace Passwordless.Service.EventLog.Models;
 
 public class EventResponse
 {
+    public EventResponse() { }
+
     public EventResponse(DateTime performedAt, string message, string performedBy, string applicationId, EventType eventType, Severity severity, string subject, string apiKeyId)
```

Comment:
If we already have `init` props, do we need the constructor at all?

Reply:
The JsonSerializer was complaining about not being able to use the constructor.